### PR TITLE
AC Automated Tests - Connection Wizard stories automation | master - WIP

### PR DESCRIPTION
**Pull request by manuferreira**
PR 1 - Ability to connect to AC from Commerce via AC token:
https://issues.liferay.com/browse/LRAC-12443 (Update macros and paths)
https://issues.liferay.com/browse/LRAC-12437 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12438 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12440 (Automation ticket)

PR 2 - Ability to connect to AC from Commerce via AC token:
https://issues.liferay.com/browse/LRAC-12439 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12457 (Update macros and paths)
https://issues.liferay.com/browse/LRAC-12458 (Update macros and paths)

PR 3 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12570 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12571 (Automation ticket)

PR 4 - Implement Property pagination during Commerce Connection:
https://issues.liferay.com/browse/LRAC-12583 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12584 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12585 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12586 (Automation ticket)

PR 5 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12572 (Automation ticket)

PR 6 - Disconnect AC Workspace:
https://issues.liferay.com/browse/LRAC-12518 (Automation ticket)

PR 7 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12574 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12575 (Automation ticket)

PR 8 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12576 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12577 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12578 (Automation ticket)

PR 9 - Ability to create new properties:
https://issues.liferay.com/browse/LRAC-12579 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12580 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12581 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12569 (Automation ticket)

PR 10 - Ability to review and change Attributes information:
https://issues.liferay.com/browse/LRAC-12679 (Update macros and paths)

PR 11 - Ability to review and change Attributes information:
https://issues.liferay.com/browse/LRAC-12676 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12677 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12678 (Automation ticket)

PR 12 - Ability to sync All Contacts and Accounts:
https://issues.liferay.com/browse/LRAC-12658 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12659 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12660 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12661 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12662 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12665 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12666 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12667 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12668 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12669 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12670 (Automation ticket)

PR 13 - Ability to sync All Contacts and Accounts:
https://issues.liferay.com/browse/LRAC-12663 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12664 (Automation ticket)

PR 14 - Ability to sync All Contacts and Accounts:
https://issues.liferay.com/browse/LRAC-12762 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12763 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12764 (Automation ticket)

PR 15 - Ability to select People Attributes:
https://issues.liferay.com/browse/LRAC-12777 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12780 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12781 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12782 (Automation ticket)

PR 16 - Test fix error while executing click
https://issues.liferay.com/browse/LRAC-12302 (Test fix ticket)
https://github.com/liferay-ac/liferay-portal/pull/271

PR 17 - Ability to select People Attributes:
https://issues.liferay.com/browse/LRAC-12778 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12779 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12783 (Automation ticket)

PR 18 - Ability to select Account Attributes:
https://issues.liferay.com/browse/LRAC-12842 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12843 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12844 (Automation ticket)

PR 19 - Ability to select Products Attributes:
https://issues.liferay.com/browse/LRAC-12901 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12903 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12904 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12905 (Automation ticket)

PR 20 - Ability to select Account Attributes:
https://issues.liferay.com/browse/LRAC-12835 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12837 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12838 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12841 (Automation ticket)

PR 21 - Ability to select Order Attributes:
https://issues.liferay.com/browse/LRAC-12909 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12910 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12911 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12913 (Automation ticket)
https://issues.liferay.com/browse/LRAC-12914 (Automation ticket)

Update:
https://issues.liferay.com/browse/LRAC-13042 (Test fix)

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase [hunk 1]
```diff
@@ -828,4 +828,37 @@ definition {
...
 	}
 
+	@description = "Story: LRAC-10867 | Automation ID: LRAC-12911 | Test Summary: Check that it is possible to search the Attribute for Orders list"
+	@priority = "4"
+	test CanSearchAttributeInOrdersList {
+		var searchedOrdersAttribute = "orderId";
+		var unsearchedOrdersAttribute = "currencyCode,orderDate";
+
+		task ("Click to select attribute for orders") {
+			ACDXPSettings.clickToSelectAttributes(attributeCategory = "Account");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase]: Hey @calvinctvo, I think this should be **Order** here. Remember to run the tests to verify that it is running without errors.

**Review by ViniLopes20 — APPROVED** [review]
Very good automations Calvin, LGTM 🎉 

Very good automations Manu and thanks for making the corrections, LGTM 🎉

**Comment by manuferreira** [comment]
Thank you all guys! :muscle:  @ViniLopes20 @calvinctvo and @joshitagaki. We created a monster :rofl: :tada:

**Comment by calvinctvo** [comment]
Thank you guys for the set up @ViniLopes20 , @manuferreira , @joshitagaki ! We finally finished! LGTM

**Review by joshitagaki — APPROVED** [review]
Great job everyone! Yea this is massive haha. Hopefully all our tests work on the first try when it goes live.

**Comment by ViniLopes20** [comment]
Hey guys, closing this ticket since I sent a new one, see: https://github.com/liferay-ac/liferay-portal/pull/327

**Pull request closed without merging**